### PR TITLE
fusuma: fix settings example

### PR DESCRIPTION
**Pull request by juboba**
### Description

The example for settings is including the top level options, which is misleading.

### Checklist

- [x] Change is backwards compatible.

- [ ] Code formatted with `./format`.

- [ ] Code tested through `nix-shell --pure tests -A run.all`.

- [ ] Test cases updated/added. See [example](https://github.com/nix-community/home-manager/commit/f3fbb50b68df20da47f9b0def5607857fcc0d021#diff-b61a6d542f9036550ba9c401c80f00ef).

- [x] Commit messages are formatted like

    ```
    {component}: {description}

    {long description}
    ```

    See [CONTRIBUTING](https://github.com/nix-community/home-manager/blob/master/docs/contributing.adoc#sec-commit-style) for more information and [recent commit messages](https://github.com/nix-community/home-manager/commits/master) for examples.

- If this PR adds a new module

  - [ ] Added myself as module maintainer. See [example](https://github.com/nix-community/home-manager/blob/068ff76a10e95820f886ac46957edcff4e44621d/modules/programs/lesspipe.nix#L6).

  - [ ] Added myself and the module files to `.github/CODEOWNERS`.

### CODE REVIEW

File: modules/services/fusuma.nix [hunk 1]
```diff
@@ -72,22 +72,18 @@ in {
...
-            interval = {
-              swipe = 0.7;
-            };
-            swipe = {
-              "3" = {
-                left = {
-                  # GNOME: Switch to left workspace
-                  command = "xdotool key ctrl+alt+Right";
-                };
+        config = {
```
Comment (sumnerevans) [inline modules/services/fusuma.nix:75]:
```suggestion
        {
```

**Review by sumnerevans — CHANGES_REQUESTED** [review]

File: modules/services/fusuma.nix (hunk 1 above)
Comment (juboba) [reply modules/services/fusuma.nix:75]: @sumnerevans totally right, thanks!

**Review by berbiche — APPROVED** [review]
LGTM!

**Review by sumnerevans — APPROVED** [review]

**Comment by sumnerevans** [comment]
@juboba please rebase on the latest `master`. The build is already fixed there.

**Comment by sumnerevans** [comment]
Thanks for the contribution!

**Pull request merged**